tests/integration/scripts/test_validate_integration.py

```diff
@@ -105,7 +105,7 @@ def test_validate_simple_graph(flow, validate, caplog):
     })
     validate(id_)
     expect = (
-        'deprecated graph items were automatically upgraded'
+        'graph items were automatically upgraded'
```

Comment:
As per my other comment, I think that the use of the word "deprecated" hints at us wanting to remove this in future.